patches/0055-Wifi-2538-RRM-rebalance-disconnects-client.patch

```diff
@@ -0,0 +1,35 @@
+From 4e8ce172f0d35424738e788faed87d26c959d82b Mon Sep 17 00:00:00 2001
+From: Ammad Rehmat <ammad.rehmat@connectus.ai>
+Date: Mon, 14 Jun 2021 15:50:59 -0400
+Subject: [PATCH] Wifi-2538 : RRM rebalance disconnects client
+
+Signed-off-by: Ammad Rehmat <ammad.rehmat@connectus.ai>
+---
+ package/system/procd/files/reload_config | 9 +++++++--
+ 1 file changed, 7 insertions(+), 2 deletions(-)
+
+diff --git a/package/system/procd/files/reload_config b/package/system/procd/files/reload_config
+index 8d1cdb1378..cb5b946054 100644
+--- a/package/system/procd/files/reload_config
++++ b/package/system/procd/files/reload_config
+@@ -5,10 +5,15 @@ for config in /etc/config/*; do
+ 	file=${config##*/}
+ 	uci show "${file##*/}" > /var/run/config.check/$file
+ done
++FILE=/tmp/wireless_changes.txt
+ MD5FILE=/var/run/config.md5
+ [ -f $MD5FILE ] && {
+-	for c in `md5sum -c $MD5FILE 2>/dev/null| grep FAILED | cut -d: -f1`; do
+-		ubus call service event "{ \"type\": \"config.change\", \"data\": { \"package\": \"$(basename $c)\" }}"
++        for c in `md5sum -c $MD5FILE 2>/dev/null| grep FAILED | cut -d: -f1`; do
++                if [ -f "$FILE" ];then
```

Comment:
This condition also needs to ensure that the change is to /etc/config/wireless.  Eg $(basename $c) needs to equal wireless.
Otherwise you are blocking every other service from updating (and calling your function too many times).